src/System.Console/ref/System.Console.cs

```diff
@@ -95,6 +97,173 @@ public enum ConsoleColor
         White = 15,
         Yellow = 14,
     }
+    public partial struct ConsoleKeyInfo
+    {
+        public ConsoleKeyInfo(char keyChar, ConsoleKey key, bool shift, bool alt, bool control) { }
+        public char KeyChar { get { return default(char); } }
+        public ConsoleKey Key { get { return default(ConsoleKey); } }
+        public ConsoleModifiers Modifiers { get { return default(ConsoleModifiers); ; } }
+    }
+    public enum ConsoleKey
+    {
+        Backspace = 0x8,
+        Tab = 0x9,
+        Clear = 0xC,
+        Enter = 0xD,
+        Pause = 0x13,
+        Escape = 0x1B,
+        Spacebar = 0x20,
+        PageUp = 0x21,
+        PageDown = 0x22,
+        End = 0x23,
+        Home = 0x24,
+        LeftArrow = 0x25,
+        UpArrow = 0x26,
+        RightArrow = 0x27,
+        DownArrow = 0x28,
+        Select = 0x29,
+        Print = 0x2A,
+        Execute = 0x2B,
+        PrintScreen = 0x2C,
+        Insert = 0x2D,
+        Delete = 0x2E,
+        Help = 0x2F,
+        D0 = 0x30,  // 0 through 9
+        D1 = 0x31,
+        D2 = 0x32,
+        D3 = 0x33,
+        D4 = 0x34,
+        D5 = 0x35,
+        D6 = 0x36,
+        D7 = 0x37,
+        D8 = 0x38,
+        D9 = 0x39,
+        A = 0x41,
+        B = 0x42,
+        C = 0x43,
+        D = 0x44,
+        E = 0x45,
+        F = 0x46,
+        G = 0x47,
+        H = 0x48,
+        I = 0x49,
+        J = 0x4A,
+        K = 0x4B,
+        L = 0x4C,
+        M = 0x4D,
+        N = 0x4E,
+        O = 0x4F,
+        P = 0x50,
+        Q = 0x51,
+        R = 0x52,
+        S = 0x53,
+        T = 0x54,
+        U = 0x55,
+        V = 0x56,
+        W = 0x57,
+        X = 0x58,
+        Y = 0x59,
+        Z = 0x5A,
+#if TODO
```

Comment:
I'm fine with leaving these out for now. I would however remove the #ifdef's and leave the entries there commented out as placeholders so folks coming along later don't put in entries that conflict with those. 

By the way please don't block on me for merging this, I would like to see it get merged today if possible.